docs/olddocs/triggers_and_events.md

```diff
@@ -84,7 +84,7 @@ Extracted entities are passed along to any triggered actions or child dialogs us
 
 When triggered, if LUIS is able to identify a city, the city name will be made available as `@city` within the triggered actions. The entity value can be used directly in expressions and LG templates, or [stored into a memory property](using_memory.md) for later use.
 
-### Regular Expression Recognizer
+### Regular expressions Recognizer
```

Comment:
The casing looks weird here.

Reply:
Yes, the casing is wired. 
These docs are outdated and archived, I am not sure if we still need to update them.

Reply:
Again, a little too focused, and the error somehow snuck buy. Regardless, I will revert it.